tests/integration_tests/functional/test_snapshot_version.py

```diff
@@ -39,38 +39,6 @@ def _create_and_start_microvm_with_net_devices(
         assert exit_code == 0
 
 
-@pytest.mark.skipif(
```

Comment:
Again, it's a nit, but I think it adds clarity and it is useful if you could mention in the commit body why we removing this (it refers to a Firecracker version we do not support any more)

Reply:
Done!